relay/sources/rudder-pkg/lib/rudder-pkg/rpkg.py

```diff
@@ -0,0 +1,151 @@
+import os, logging, re, json
+from pkg_resources import parse_version
+
+import rudderPkgUtils as utils
+
+"""
+    Versions can contains the words "-SNAPSHOT" at the end, this should not be a problem at the moment since it is only present in nightly.
+    Moreover, we should not have to compare a nightly version to a release one.
```

Comment:
What about tests ? We should be able to compare then
x-SNAPSHOT > x
x-SNAPSHOT < y when y < x